src/Common/Histogram.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <base/types.h>
+#include <Common/SipHash.h>
+
+#include <atomic>
+#include <memory>
+#include <mutex>
+#include <vector>
+
+namespace Histogram
```

Comment:
For consistency with the project this should be `DB` first, then `Histogram` if you need a nested namespace.